code/style.css

```diff
@@ -1,4 +1,401 @@
+  z-index: 2;
+}
+
+.mobile-wrap-item1 {
+  width: 8rem;
+  text-align: left;
+}
+
+.mobile-wrap-item2 {
+  width: 8rem;
+  text-align: center;
+}
+
+.mobile-wrap-item3 {
+  width: 8rem;
+  text-align: right;
+}
+
+.logo-mobile {
+  height: 3rem;
+  padding-top: 0.5rem;
+}
+
+.menu-icon {
+  font-size: 2rem;
+  padding-top: 0.5rem;
+  color: #82171A;
+  line-height: 2rem;
+}
+
+.mobile-wrap nav {
+  position: absolute;
+  background-color: #82171A;
+  top: 4.5rem;
+  left: 0;
+  width: 50%;
+  z-index: 1;
+}
+
+.mobile-wrap nav ul {
+  list-style-type: none;
+  max-height: 0;
+  overflow: hidden;
+  transition: all 0.5s;
+}
+
+.mobile-wrap nav ul li {
+  padding: 0 0.5rem;
+}
+
+.mobile-wrap nav ul li a {
+  display: inline-block;
+  padding: 0.75rem;
+  color: white;
+  text-decoration: none;
+}
+
+#checkbox {
+  display: none;
+}
+
+#checkbox:checked~nav ul {
+  max-height: 18rem;
+  padding: 1rem 0;
+  transition: all 0.5s;
+}
+
+@media (min-width: 1025px) {
+  header.mobile {
+    display: none;
+  }
+}
+/* END Header with logo, links and date only visible on mobile (max 1024px) */
+
+/* START Header with logo, links and date only visible on desktop (min 1025px) */
+header.desktop {
+  display: none;
+}
+
+header.desktop img {
+  width: 12rem;
+}
+
+.navbar, time {
+  font-family: 'Raleway', sans-serif;
+  font-size: 1.2rem;
+}
+
+.navbar {
+  color: black;
+}
+
+time {
+  color: #82171A;
+}
+
+@media (min-width: 1025px) {
+  header.desktop {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border: solid #82171A;
+    border-width: 1rem 0;
+    margin-bottom: 1rem;
+    padding: 0 6rem;
+  }
+}
+
+/* Different colors when hovering over items of navbar */
+@media (min-width: 1025px) {
+  .desktop .navbar:hover {
+    -webkit-text-decoration-line: underline;
+    text-decoration-line: underline;
+    text-decoration-thickness: 0.4rem;
+    text-underline-offset: 0.4rem;
+  }
+
+  .desktop .news:hover {
+    -webkit-text-decoration-color: rgb(255, 165, 0, 0.50);
+    text-decoration-color: rgb(255, 165, 0, 0.50);
+  }
+
+  .desktop .law:hover {
+    -webkit-text-decoration-color: rgb(255, 255, 0, 0.50);
+    text-decoration-color: rgb(255, 255, 0, 0.50);
+  }
+
+  .desktop .travel:hover {
+    -webkit-text-decoration-color: rgb(0, 128, 0, 0.50);
+    text-decoration-color: rgb(0, 128, 0, 0.50);
+  }
+
+  .desktop .sports:hover {
+    -webkit-text-decoration-color: rgb(0, 0, 255, 0.50);
+    text-decoration-color: rgb(0, 0, 255, 0.50);
+  }
+
+  .desktop .society:hover {
+    -webkit-text-decoration-color: rgb(75, 0, 130, 0.50);
+    text-decoration-color: rgb(75, 0, 130, 0.50);
+  }
+
+  .desktop .weather:hover {
+    -webkit-text-decoration-color: rgb(238, 130, 238, 0.50);
+    text-decoration-color: rgb(238, 130, 238, 0.50);
+  }
+}
+/* END Header with logo, links and date only visible on desktop (min 1025px) */
+
+/* START Header with ticker always visible */
+@-webkit-keyframes ticker {
```

Comment:
Well done on the ticker! You get an extra bonus star, for making it look like a real news site with the latest news coming in.